integration/go.mod

```diff
@@ -359,3 +359,6 @@ require (
 replace github.com/onflow/flow-go => ../
 
 replace github.com/onflow/flow-go/insecure => ../insecure
+
+// TODO: remove it when https://github.com/onflow/flow-emulator/pull/724 merged
+replace github.com/onflow/flow-emulator v1.0.0-preview.41.0.20240829134601-0be55d6970b5 => github.com/AndriiDiachuk/flow-emulator v0.0.0-20240903140813-122b79780f82
```

Comment:
please remove this replace and update the `flow-emulator` version to be the commit from your emulator PR